src/services/storage.service.js

```diff
@@ -40,7 +40,7 @@ function createStorageService() {
 
   async function upload(fileBlob, { onSuccess, onError, onProgress }) {
     try {
-      const fileDid = await storage.upload(fileBlob, onProgress);
+      const fileDid = await storage.upload(fileBlob, { onProgress });
```

Comment:
Just a question for now: This method is doing little more than wrapping the `storage.upload` call. Would it be better if we remove these service methods and expose only the storage instance?

@shrinathprabhu @karthik-durai

Reply:
I guess this is fine, the service file gives us all the functions needed to interact with storage service